.github/workflows/ci.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           --exclude '[^\w]todo[^\w]?'
           --exclude 'https://(www\.|old\.)?reddit\.com'
           --exclude 'https://www.patreon.com'
+          --exclude 'https://gamedev.social'
```

Comment:
Wow, I believe this could lead to problems in the future. So we won't be able to check for broken links on this (literally this) host.

Reply:
I realize that right now the resource is not available. It's probably worth returning this check when everything is back to normal.

Reply:
@janhohenheim 😱

Reply:
Yeah, I hope to remove that entry later today. But at the end of the day, a broken Mastodon link is also not that big of an issue :)

Reply:
Oh yes, I agree. In general, I confused `gamedev.social` with `rust-gamedev.github.io` 🤦🏻‍♂️ and thought that remove check for self-refs is not a good idea. Sorry, you're totally right.

Reply:
Aaah, I see :D Yeah, that be bad, haha